stdlib/REPL/test/docview.jl

```diff
@@ -0,0 +1,13 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+import REPL
+
+@test startswith(let buf = IOBuffer()
+        Core.eval(Main, REPL.helpmode(buf, "α"))
+        String(take!(buf))
+    end, "\"α\" can be typed by \\alpha<tab>\n")
+
+@test startswith(let buf = IOBuffer()
+        Core.eval(Main, REPL.helpmode(buf, "🐨"))
+        String(take!(buf))
+    end, "\"🐨\" can be typed by \\coala<tab>\n")
```

Comment:
"koala", not "coala"?

Reply:
And should also have colons:  `\:koala:<tab>`

Reply:
Oh my bad. Fixed! :smile: